client/src/components/User/index.js

```diff
@@ -13,28 +13,19 @@ const User = (props) => {
 
   return (
     <div className="user_container">
-      <UserAvatar filename={user.avatar} />
+      <div className="avatar">
+        <UserAvatar filename={user.avatar} />
+      </div>
+
       <div className="data">
         <div>
-          <span>
-            Name:
-          </span>
-          {' '}
-          {user.name}
+          <span>Name:</span>{' '}{user.name}
```

Comment:
I am not sure if it is really needed (the space in brackets), but ESlint hardly "suggested" me to do it this way